homeassistant/components/xiaomi_miio/translations/ca.json

```diff
@@ -10,7 +10,7 @@
         "error": {
             "cannot_connect": "Ha fallat la connexi\u00f3",
             "cloud_credentials_incomplete": "Credencials del n\u00favol incompletes, introdueix el nom d'usuari, la contrasenya i el pa\u00eds",
-            "cloud_login_error": "No s'ha pogut iniciar sessi\u00f3 a Xioami Miio Cloud, comprova les credencials.",
+            "cloud_login_error": "No s'ha pogut iniciar sessi\u00f3 a Xiaomi Miio Cloud, comprova les credencials.",
```

Comment:
Please don't change any translation files. They will be updated via Lokalise. The change in `strings.json` is enough.

Reply:
Didn't know, thx for the heads-up!